Composer/packages/client/src/pages/setting/dialog-settings/index.tsx

```diff
@@ -63,12 +55,13 @@ export const DialogSettings = () => {
     }
   };
 
-  const handleChange = (result, commit) => {
-    setValue(result);
-    if (commit || !absHosted) {
-      saveChangeResult(result);
-    }
-  };
+  const handleChange = useMemo(
+    () =>
+      debounce((result: any) => {
+        saveChangeResult(result);
+      }, 200),
+    [visibleSettings]
+  );
```

Comment:
won't this change every time `visibleSettings` changes? This will produce a new debounced change handler.

Reply:
I change it to projectId.